changelog.d/20763_loki_sink_drop_invalid_timestamp_event.fix.md

```diff
@@ -0,0 +1,3 @@
+Loki sink now drops events with non-processable timestamps.
```

Comment:
```suggestion
Loki sink now drops events with non-parsable timestamps.
```

I think `parsable` is actually the correct spelling, but what you have here seems fine to me too. Thanks for adding the changelog!